src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -505,6 +510,72 @@ def _test_log_file_mgmt(self, resource_group, server, database_engine):
+                 .format(vnet, resource_group, loc, subnet),
+                 checks=self.check('length(newVNet.subnets)', 1))
+        self.cmd('network vnet subnet update -n {} --vnet-name {} -g {} '
+                 '--disable-private-endpoint-network-policies true'
+                 .format(subnet, vnet, resource_group),
+                 checks=self.check('privateEndpointNetworkPolicies', 'Disabled'))
+
+        # Get Server Id and Group Id
+        result = self.cmd('{} server show -g {} -n {}'
+                          .format(database_engine, resource_group, server)).get_output_in_json()
+        server_id = result['id']
+        result = self.cmd('{} server private-link-resource list -g {} -s {}'
+                          .format(database_engine, resource_group, server)).get_output_in_json()
+        group_id = result[0]['properties']['groupId']
+
+        # Create a private endpoint connection
+        private_endpoint = self.cmd('network private-endpoint create -g {} -n {} --vnet-name {} --subnet {} -l {} '
+                                    '--connection-name {} --private-connection-resource-id {} '
+                                    '--group-ids {}'
+                                    .format(resource_group, pe, vnet, subnet, loc, pe_connection, server_id, group_id)).get_output_in_json()
+        self.assertEqual(private_endpoint['name'], pe)
+        self.assertEqual(private_endpoint['privateLinkServiceConnections'][0]['name'], pe_connection)
+        self.assertEqual(private_endpoint['privateLinkServiceConnections'][0]['privateLinkServiceConnectionState']['status'], 'Approved')
+        self.assertEqual(private_endpoint['privateLinkServiceConnections'][0]['provisioningState'], 'Succeeded')
+        self.assertEqual(private_endpoint['privateLinkServiceConnections'][0]['groupIds'][0], group_id)
+
+        # Get Private Endpoint Connection Name and Id
+        api_version = '2018-06-01' if database_engine == 'mariadb' else '2017-12-01'
+        result = self.cmd('az rest --method get --uri https://management.azure.com{}?api-version={}'
+                          .format(server_id, api_version, server)).get_output_in_json()
+        self.assertEqual(len(result['properties']['privateEndpointConnections']), 1)
+        self.assertEqual(result['properties']['privateEndpointConnections'][0]['properties']['privateLinkServiceConnectionState']['status'],
+                         'Approved')
+        server_pec_id = result['properties']['privateEndpointConnections'][0]['id']
+        result = parse_proxy_resource_id(server_pec_id)
+        server_pec_name = result['child_name_1']
+
+        self.cmd('{} server private-endpoint-connection show --server-name {} -g {} --name {}'
+                 .format(database_engine, server, resource_group, server_pec_name),
+                 checks=self.check('id', server_pec_id))
+
+        with self.assertRaisesRegexp(CloudError, 'Private Endpoint Connection Status is not Pending'):
```

Comment:
Once the connection is approved, it's cannot be reject/approve anymore?

Reply:
yes.